srtcore/api.h

```diff
@@ -462,6 +462,9 @@ class CUDTUnited
 
     CUDTSocket* locateAcquireSocket(SRTSOCKET u, ErrorHandling erh = ERH_RETURN);
     bool acquireSocket(CUDTSocket* s);
+    bool startGarbageCollector();
```

Comment:
If these functions require `m_InitLock` to be locked, please add appropriate TSA annotation.